### PR TITLE
more draggable modals

### PR DESCRIPTION
**Pull request by mcfiredrill**

### CODE REVIEW

**Comment by dudethatbe** [comment]
huh for some reason when I click on the GIF button, the dialog appears in the right-corner, and almost immediately turns into the Congratulation :smile: message ?
image

File: app/components/datafruits-chat.js [hunk 1]
```diff
@@ -77,7 +77,8 @@ export default class DatafruitsChat extends Component {
   }
 
   @tracked
-  virusTimer = 500;
+  virusTimer = 3;
```
Comment (mcfiredrill) [inline app/components/datafruits-chat.js:80]:
```suggestion
  virusTimer = 3000;
```

**Comment by mcfiredrill** [comment]
> dialog appears in the right-corner
Yeah the centering could use some work

**Comment by mcfiredrill** [comment]
Multiple badge viewer modals working

**Review by dudethatbe — APPROVED** [review]

**Pull request merged**